src/librustc_target/spec/abi.rs

```diff
@@ -11,6 +11,14 @@ pub enum Abi {
     // N.B., this ordering MUST match the AbiDatas array below.
     // (This is ensured by the test indices_are_correct().)
 
+    // Multiplatform / generic ABIs
+    //
+    // These two come first because every time we add a new ABI, we
+    // have to re-bless all the hashing tests. These two are used in
+    // many places, so giving them stable values reduces test churn.
+    Rust,
+    C,
```

Comment:
If the goal is stable values, what about explicitly assigning these values?

Reply:
My original idea was to _just_ number them, but that ran into [`E0081`](https://doc.rust-lang.org/error-index.html#E0081). Since I had to reorder it anyway, I backed away from adding the explicit numbers in case future readers thought that those values _meant_ something specific.

If you think that adding explicit values would be beneficial, I'm OK to add them as well.

Reply:
Yes, I feel like with a comment that says "we want the values to be fixed", it would be useful to the reader to also actually explicitly fix those values.